CONTRIBUTING.md

```diff
@@ -1,63 +1,67 @@
 ## Reporting Bugs
 
-Please file bugs in the [GitHub Issue
-Tracker](https://github.com/syncthing/syncthing/issues). Include at
-least the following:
+Please file bugs in the [GitHub Issue Tracker][issue tracker].  
+Include at least the following:
 
- - What happened
+* What happened.
 
- - What did you expect to happen instead of what *did* happen, if it's
-   not crazy obvious
+* What you expected should happen instead of what *did* happen. 
+  (Unless it's highly obvious.)
 
- - What operating system, operating system version and version of
-   Syncthing you are running
+* The operating system name, operating system version, 
+  and version of Syncthing, which you are running.
 
- - The same for other connected devices, where relevant
+* The same for other connected devices, where relevant.
 
- - Screenshot if the issue concerns something visible in the GUI
+* Screenshot(s) if the issue concerns something visible in the GUI.
 
- - Console log entries, where possible and relevant
+* Console log entries, where possible and relevant.
 
 If you're not sure whether something is relevant, erring on the side of
-too much information will never get you yelled at. :)
+too much information will never get you yelled at. :-)
 
 ## Contributing Translations
 
-All translations are done via
-[Transifex](https://www.transifex.com/projects/p/syncthing/). If you
-wish to contribute to a translation, just head over there and sign up.
-Before every release, the language resources are updated from the
-latest info on Transifex.
+All translations are done via [Transifex][transifex]. 
+If you wish to contribute to a translation, just head over there and sign up.
+Before every release, the language resources are updated from the latest info 
+on Transifex.
 
 ## Contributing Code
 
-Every contribution is welcome. If you want to contribute but are unsure
-where to start, any open issues are fair game! See the [Contribution
-Guidelines](https://docs.syncthing.net/dev/contributing.html) for the full
-story on committing code.
+Every contribution is welcome. 
+If you want to contribute but are unsure where to start, 
+any open issues are fair game! 
+See the [Contribution Guidelines][contributing] 
+for the full story on committing code.
```

Comment:
I find this harder to read with such short lines.  Why not just apply word-wrapping at a constant column (80 suggested)?

Still we should not forget this is probably one of the least important source files in the whole repo, because it just provides the template / guidelines for creating a PR.  Almost nobody will look at it other than in rendered HTML form.  So I politely suggest you should focus on more important things regarding Syncthing development if you want to help out.  This kind of change costs precious time from reviewers and if it changes too many unimportant things along the road, things start to get confusing.